AnyServe/AnyServe/Controllers/FileUploadController.cs

```diff
@@ -0,0 +1,178 @@
+            string pathToFiles = Path.Combine(_appEnvironment.WebRootPath, _helper.directoryName);
+
+            return Ok(AllFilesInWebRootPath(pathToFiles));
+        }
+
+        // Single file upload
+        [HttpPost("UploadFile")]
+        public async Task<IActionResult> UploadFile([FromForm] IFormFile uploadedFile)
+        {
+            if (uploadedFile != null && Path.GetExtension(uploadedFile.FileName) != null)
+            {
+                var uploadsRootFolder = Path.Combine(_appEnvironment.WebRootPath, _helper.directoryName);
+
+                if (!Directory.Exists(uploadsRootFolder))
+                    Directory.CreateDirectory(uploadsRootFolder);
+
+                Guid id = Guid.NewGuid();
+
+                //new unique file name
+                string fileName = id + Path.GetExtension(uploadedFile.FileName);
+                string fullFilePath = uploadsRootFolder + fileName;
+
+                // save file to folder Files in catalog wwwroot
+                using (var fileStream = new FileStream(fullFilePath, FileMode.Create))
+                {
+                    await uploadedFile.CopyToAsync(fileStream);
+                }
+
+                //add following instance to db
+                FileModel file = new FileModel { Name = fileName, Path = fullFilePath, Id = id, OriginalName = uploadedFile.FileName };
+             
+                return Ok(new FileModelResponse(file));
+
+            }
+
+
+            //TODO
+            // Need to deside what return then file is not in Form
+            return BadRequest("Unable to upload file");
+        }
+
+
+        // Multiple files upload
+        [HttpPost("UploadFiles")]
+        public async Task<IActionResult> UploadFiles(List<IFormFile> uploads)
+        {
+            if (uploads != null)
+            {
+                List<FileModel> fileToDB = new List<FileModel>();
+                List<FileModelResponse> filesResponse = new List<FileModelResponse>();
+
+                //Path for wwwroot\Files
+                var uploadsRootFolder = Path.Combine(_appEnvironment.WebRootPath, _helper.directoryName);
+
+                if (!Directory.Exists(uploadsRootFolder))
+                    Directory.CreateDirectory(uploadsRootFolder);
+
+                foreach (var uploadedFile in uploads)
+                {
+                    Guid id = Guid.NewGuid();
+
+                    //new unique file name
+                    string fileName = id + Path.GetExtension(uploadedFile.FileName);
+                    string fullFilePath = uploadsRootFolder + fileName;
+
+                    // save file to folder Files to catalog wwwroot\Files
+                    using (var fileStream = new FileStream(fullFilePath, FileMode.Create))
+                    {
+                        await uploadedFile.CopyToAsync(fileStream);
+
+                        var newFile = new FileModel { Name = fileName, Path = fullFilePath, Id = id, OriginalName = uploadedFile.FileName };
```

Comment:
As you can see upload files is like many times upload one file. Good place to refactor, move uploading logic to different method and use it when needed. Ok for now